src/layouts/index.js

```diff
@@ -23,7 +24,12 @@ import 'glamor/reset';
 import 'css/reset.css';
 import 'css/algolia.css';
 
-class Template extends Component {
+type Props = {
+  children: any, // I don't think this is a Node type
```

Comment:
This is a Function. (Also a built-in Gatsby thing): https://www.gatsbyjs.org/docs/building-with-components/#layout-components